docs/source/models/causes/neonatal/preterm_birth.rst

```diff
@@ -154,17 +154,31 @@ Data Tables
 The Preterm Birth model requires only the probability of death (aka "mortality risk") for use
 in the decision graph. This will be computed from the overall neonatal mortality risk and the cause-specific mortality fraction.
 
-Since this is a PAF-of-one cause, the calculation must take into account the "structural zeros" representing no mortality risk for simulants withe a gestational age above 37 weeks.
+Since this is a PAF-of-one cause, the calculation must take into account the "structural zeros" representing no mortality risk for simulants with a gestational age of 38 or more weeks.
```

Comment:
```suggestion
Since this is a PAF-of-one cause, the calculation must take into account the "structural zeros" representing no mortality risk for simulants with a gestational age of 37 or more weeks.
```
Again, I thought "term" means $\ge 37$ weeks...